scripts/Connector/Connector_SMD_single_row_plus_mounting_pad/conn_hirose.yaml

```diff
@@ -2,6 +2,29 @@ group_definitions:
     manufacturer: 'Hirose'
 
 device_definition:
+    DF3EA:
+        series: ''
+        mpn_format_string: 'DF13EA-{pincount:02d}P-2H'
+        orientation: 'H'
+        datasheet: 'https://www.hirose.com/product/document?clcode=CL0543-0332-0-51&productname=DF3EA-5P-2H(51)&series=DF3&documenttype=2DDrawing&lang=en&documentid=0001163317'
+        pinrange: ['list', [2,3,4,5,6,7,8,9,10,11,12,13,14,15]]
+        text_inside_pos: 'center'
+        pitch: 2
+        pad1_position: 'top-left'
+        mounting_pad_size: [2.3, 3.3]
+        # x position mounting inner mounting pad edge relative to nearest pad center
+        center_pad_to_mounting_pad_edge: 3.37
+        # y dimensions for pad given relative to mounting pad edge
+        rel_pad_y_outside_edge: 8.9
+        rel_pad_y_inside_edge: 5.8
+        pad_size_x: 1.2
+        # y position for body edge relative to mounting pad edge (positive -> body extends outside bounding box)
+        rel_body_edge_y: 0.4
```

Comment:
I think 0.45 would be the correct value
y1: body front to front of mounting pin
y2: size of mounting pin
y3: body front to center of mounting pin
y4: size of pad for mounting pins

```
y1 + y2/2 = 1 + 2.2/2 = 2.1 = y3
rel_body_edge_y = y3 - y4/2 = 2.1 - 3.3/2 = 0.45
```